.github/workflows/helm-chart-ci.yaml

```diff
@@ -151,9 +151,15 @@ jobs:
 
       - name: Run chart-testing (install)
         run: |
+          [ "$VALUES" != "default" ] && kubectl create namespace "$VALUES"
+          [ -x .github/tests/$VALUES/pre-install.sh ] && .github/tests/$VALUES/pre-install.sh
           ct install --debug \
+            --namespace $VALUES \
             --target-branch ${{ github.base_ref }} \
             --exclude-deprecated \
             ${{ (matrix.values != 'default' && '--helm-extra-set-args "--values=.github/tests/$VALUES/values.yaml"') || '' }}
+          res=$?
+          [ -x .github/tests/$VALUES/post-install.sh ] && .github/tests/$VALUES/post-install.sh
+          exit $res
```

Comment:
I would implement this using a `trap` to make it less errorprone

https://medium.com/@dirk.avery/the-bash-trap-trap-ce6083f36700

Reply:
Done